budgea_openapi.json

```diff
@@ -26878,6 +26881,28 @@
         }
       }
     },
+    "PaymentAccount": {
+      "type": "object",
+      "properties": {
+        "label": {
+          "type": "string",
+          "description": "Display name of the payment account.",
+          "nullable": true
```

Comment:
I see label as `required: True` is it nullable ?

Reply:
On the "PaymentAccount object" its type is "String or null" 🤔

Reply:
You are right I just seen as required in https://docs.budget-insight.com/reference/payments#paymentaccountrequest-object and I was a bit surprised :)